### ISSUE TITLE
Syrup Bomb, White Smoke/Clear Body and Clear Amulet interaction is wrong 

### ISSUE THREAD

**Issue by AlexOn1ine** [smogon/pokemon-showdown#9845]
In SV if a Pokemon with White Smoke, Clear Body or Clear Amulet is covered with Sticky Syrup there is a message when the Speed drop should happen.
![image](https://github.com/smogon/pokemon-showdown/assets/93446519/6336dcdc-1575-4b3a-8a15-afd2e9738257)
No image for Clear Amulet but this is the message:
```
The effects of {mon} Clear Amulet prevent its stats from being lowered!
```
On showdown there is no such message.
![image](https://github.com/smogon/pokemon-showdown/assets/93446519/cc11523d-1ad8-4b19-aaa8-448b32e53a0c)

**Comment by urkerab** [comment]
This replay excerpt shows no sticky syrup as syrup bomb missed...

**Comment by AlexOn1ine** [comment]
> This replay excerpt shows no sticky syrup as syrup bomb missed...

yeah, sorry didn't pay attention lol
Here is the correct one
![image](https://github.com/smogon/pokemon-showdown/assets/93446519/ca037267-0270-455b-b8a8-5aa230a8bcf6)

**Comment by Karthik99999** [comment]
I incorrectly assumed that the messages didn't show because Octolock (another move that drops stats at the end of the turn) didn't show these messages last gen. I will check Octolock in this gen and fix this

**Comment by AlexOn1ine** [comment]
Fixed in  #9847
